### ISSUE TITLE
Font certificates

### ISSUE THREAD

**Issue by ritesh94** [googlesamples/android-EmojiCompat#19]
From where do we get the fonts certificates?
Can we use the certificates from this project sample?

**Comment by siyamed** [comment]
Yes the certificates in the example should be for google font provider. 

Android Studio will add the correct signature for you. Please check steps in here: https://developer.android.com/guide/topics/ui/look-and-feel/downloadable-fonts#via-android-studio
